### DISCUSSION TITLE
Vercel Secret Manager

### DISCUSSION THREAD

**Discussion by nirmalrepo** [vercel/community#5436] [Help] [answered]
### Summary

Hi,

I would like to know about what everyone is using to store there secrets. I came across Doppler and HarshCrop. Would love to see if anyone has any suggestions 

### Steps to Reproduce

N/A

**Comment by leerob** [comment, marked as answer]
Hey! We now support storings secrets as environment variables.

https://vercel.com/changelog/sensitive-environment-variables-are-now-available
